package.json

```diff
@@ -210,9 +210,11 @@
     "electron-debug": "^3.2.0",
     "electron-log": "^4.4.6",
     "electron-updater": "^4.6.5",
+    "express": "^4.18.1",
     "fluent-ffmpeg": "^2.1.2",
     "get-audio-duration": "^3.1.0",
     "history": "^5.3.0",
+    "nodemon": "^2.0.16",
```

Comment:
nodemon is a dev dependency, Could you move it from `dependencies` to `devDependencies`